msteams-platform/feedback.md

```diff
@@ -2,28 +2,38 @@
+| **Samples** |             Submit issues with samples to respective [Microsoft Teams Samples](/microsoftteams/platform/tutorials/code-samples) repository.                                                                                                                                                                                            |
+| **Features suggestions**             |      Make your feature's suggestions on the [Microsoft Teams feedback portal on UserVoice](https://microsoftteams.uservoice.com/forums/555103-public-preview/category/182881-developer-platform).                                                                                                                                                            |
+|        **Documentation issues**        |                                                                                                                                                                      Choose `This page` option in the **Submit and view feedback for** section, available at bottom of respective documentation to open an [issue](https://github.com/MicrosoftDocs/msteams-docs/issues) on the Microsoft Teams documentation GitHub repository.                                                                                                                                                                      |
+|       **Documentation updates**        | Select the **Edit** link in an article and submit a pull request to the [Microsoft Teams documentation GitHub](https://github.com/MicrosoftDocs/msteams-docs) repository.                                                                                                                                                                      |
+|       **Community chat group**        | Select **SIGN IN TO START TALKING** to join the informal community chat room for [Microsoft Teams app developers](https://gitter.im/OfficeDev/MicrosoftTeamsAppDev) if you are logging in for the first time. Select **JOIN ROOM**, if you are already logged in. |
+|          **General questions**         |          General questions about the Microsoft Teams developer platform must be sent to [Microsoft Teams Community Help](mailto:microsoftteamsdev@microsoft.com). We encourage posting questions on channels mentioned and use email only if none of the above are applicable.                                                                                                                                                                          |
 
 ## Product Support channels
-|            **Support type**            |               **Contact**                                                                                  |
+Support channels to submit your Teams product related questions:
```

Comment:
The following table lists the support channels to submit Teams product related queries: